resources/js/views/home/Community/StatsSection.vue

```diff
@@ -0,0 +1,182 @@
+                    <div class="total has-text-weight-bold">
+                        <number
+                            :from="0"
+                            :to="stats.litterTagsPerMonth"
+                            :duration="2"
+                            :delay="0"
+                            easing="Power1.easeOut"
+                            :format="commas"
+                        />
+                    </div>
+                    <div class="is-size-5">{{ $t('home.community.litter-tags-last-month') }}</div>
+                </div>
+            </div>
+            <div class="charts mt-6">
+                <div class="chart">
+                    <StatsChart :chart-data="yearlyStats" :options="options" />
+                </div>
+            </div>
+        </div>
+    </section>
+</template>
+<script>
+import StatsChart from './StatsChart.js';
+
+export default {
+    name: 'StatsSection',
+    components: {StatsChart},
+    data ()
+    {
+        return {
+            yearlyStats: {},
+            options: {
+                aspectRatio: 3,
+                maintainAspectRatio: false,
+                legend: {labels: {fontColor: 'whitesmoke'}},
+                scales: {
+                    xAxes: [{gridLines: {display: false}, ticks: {fontColor: 'whitesmoke'}}],
+                    yAxes: [{gridLines: {display: false}, ticks: {fontColor: 'whitesmoke'}}],
+                }
+            }
+        };
+    },
+    computed: {
+        stats() {
+            return this.$store.state.community;
+        }
+    },
+    async mounted ()
+    {
+        await this.fillData();
+    },
+    methods: {
+        async fillData ()
+        {
+            await this.$store.dispatch('GET_STATS');
+
+            this.yearlyStats = {
+                labels: this.stats.statsByMonth.periods,
+                datasets: [
+                    {
+                        label: this.$i18n.t('home.community.photos-every-month-label'),
+                        borderColor: '#1DD3B0',
+                        borderWidth: 3,
+                        pointBackgroundColor: '#008080',
+                        pointBorderColor: '#008080',
+                        backgroundColor: 'transparent',
+                        data: this.stats.statsByMonth.photosByMonth
+                    },
+                    {
+                        label: this.$i18n.t('home.community.users-every-month-label'),
+                        borderColor: '#c2f970',
+                        borderWidth: 3,
+                        pointBackgroundColor: '#008080',
+                        pointBorderColor: '#008080',
+                        backgroundColor: 'transparent',
+                        data: this.stats.statsByMonth.usersByMonth
+                    }
+                ]
+            };
+        },
+
+        /**
+         * Format number value
+         */
+        commas (n)
+        {
+            return parseInt(n).toLocaleString();
```

Comment:
We actually have a global filter helper for this on app.js, you can do 

```
{{ n | 'commas' }}
```

Reply:
True, but we're using the `<number />` component above, hence the method declared here.